### DISCUSSION TITLE
Failed to Execute [Winapps]

### DISCUSSION THREAD

**Discussion by chefpenguino** [winapps-org/winapps#325] [Support] [answered]
![image](https://github.com/user-attachments/assets/7ddddd46-bd35-4b1a-8463-9cbb22ea58bf)

I installed winapps as user, and it worked for the most part - but I get this error when i try opening any shortcut that has been made by winapps. 
I've also noticed that typing 'winapps' in the terminal returns 'command not found'.
Although, when i go into my terminal and manually run `/home/user/.local/bin/winapps/winapps Photoshop', it works without any problems. 

I assume this is some sort of symlink error? How do i fix this?

**Comment by oskardotglobal** [comment]
remove the `~/.local/bin/winapps` directory and re-run the setup

**Comment by chefpenguino** [comment]
> remove the `~/.local/bin/winapps` directory and re-run the setup

Still doesn't work, now i get another error.
![image](https://github.com/user-attachments/assets/e29f35f4-d190-4a94-999a-154c73199294)

The VM is present, so this should not occur.
![image](https://github.com/user-attachments/assets/b9939d1d-b819-414c-8b00-85428aeeb8f2)

**Comment by oskardotglobal** [comment, marked as answer]
Did you properly set the LIBVIRT_URL env var?

**Reply by chefpenguino** [reply]
My bad, fixed ^
